install_config/configuring_sdn.adoc

```diff
@@ -289,3 +303,43 @@ flannel_interface=eth0
 . Optionally, you can specify the interface to use for inter-host communication
 using the `flannel_interface` variable. Without this variable, the
 {product-title} installation uses the default interface.
+
+NOTE: In order to provide custom networking CIDR for pods and services using
+flannel, there is a bugzilla being invested by Red Hat Bugzilla ID 1473858 at
+the time of writing this paper.
+
+If using a different network interface (eth1), remember to configure the
+interface to start at boot time as:
+*_/etc/sysconfig/network-scripts/ifcfg-eth1_* file:
+----
+DEVICE=eth1
+TYPE=Ethernet
+BOOTPROTO=dhcp
+ONBOOT=yes
+DEFTROUTE=no
+PEERDNS=no
+----
+
+=== Flannel iptables rules
+In order to enable flannel, a set of iptables rules should be added as well
+after the {product-title} installation has been completed.
+
+On every {product-title} node:
+
+----
+iptables -A DOCKER -p all -j ACCEPT
+iptables -t nat -A POSTROUTING -o eth1 -j MASQUERADE
+----
+
+To persist those changes in the *_/etc/sysconfig/iptables_* use the following
+snippet on every node:
+
+----
+cp /etc/sysconfig/iptables{,.orig}
+tac /etc/sysconfig/iptables.orig | sed -e '0,/:DOCKER -/ s/:DOCKER -/:DOCKER ACCEPT/' | awk '"\!"p && /POSTROUTING/{print \"-A POSTROUTING -o eth1 -j MASQUERADE\"; p=1} 1' | tac > /etc/sysconfig/iptables
```

Comment:
this command 'as is' runs only if wrapped with `sh -c " ... "`. Additionally, if there was no original `/etc/sysconfig/iptables` file existed, it fails to create an expected `/etc/sysconfig/iptables`

Reply:
My bad, this was just copy & pasting the ansible command from here https://access.redhat.com/documentation/en-us/reference_architectures/2017/html-single/deploying_and_managing_red_hat_openshift_container_platform_3.6_on_red_hat_openstack_platform_10/#post_installation_tasks

Regarding the iptables missing file, that's why the cp should be done before